src/Benda95280/MyEntities/MyEntities.php

```diff
@@ -161,6 +172,28 @@ public static function getPlayerHeadItem(string $name, string $nameFinal, array
         return $item;
     }
 
+    /**
+     * @param HeadProperties $properties
+     * @return Item
+     * @throws \InvalidArgumentException
+     */
+    public static function getPlayerHeadItem2(HeadProperties $properties): Item
```

Comment:
Could it be possible to have one function instead, to get item ?
(Function seems to be near the same)